webapp/app/js/models/resultsets_store.js

```diff
@@ -35,9 +35,7 @@ treeherder.factory('ThResultSetModel', [
 
     var resultSetPollers = {};
     var resultSetPollInterval = 60000;
-    var jobPollInterval = 90000;
-    var pollDelayMin = 1000;
-    var pollDelayMax = 60000;
+    var jobPollInterval = 60000;
```

Comment:
Dropping from 90s to 60s interval is great - one of the common complaints is that data takes longer to appear in Treeherder - this is one step to improving that :-)